packages/react-components/src/organisms/UserTeamsTabbedCard.tsx

```diff
@@ -0,0 +1,173 @@
+import { TeamRole, UserTeam } from '@asap-hub/model';
+import { network } from '@asap-hub/routing';
+import { css } from '@emotion/react';
+import React, { ComponentProps, Fragment } from 'react';
+import { Divider, Link, Paragraph } from '../atoms';
+import { inactiveBadgeIcon } from '../icons';
+import { TabbedCard } from '../molecules';
+import { perRem, rem, tabletScreen } from '../pixels';
+import { splitListBy } from '../utils';
+import { formatDateToTimezone } from '../date';
+
+const MAX_TEAMS = 5;
+
+const inactiveBadgeStyles = css({
+  lineHeight: `${18 / perRem}em`,
+  verticalAlign: 'middle',
+  marginLeft: `${8 / perRem}em`,
+});
+
+const listItemStyle = css({
+  display: 'grid',
+
+  gridTemplateColumns: '1fr',
+  gridTemplateRows: '1fr 1fr',
+  rowGap: rem(12),
+
+  [`@media (min-width: ${tabletScreen.min}px)`]: {
+    gridAutoFlow: 'column',
+    gridTemplateColumns: '1fr 1fr',
+
+    '&:not(:first-of-type)': {
+      gridTemplateRows: '1fr',
+    },
+
+    [`&:not(:first-of-type) > :nth-of-type(odd)`]: {
+      display: 'none',
+    },
+  },
+});
+
+const containerStyle = css({
+  display: 'grid',
+
+  columnGap: rem(12),
+
+  margin: 0,
+  marginTop: `${24 / perRem}em`,
+
+  padding: 0,
+  listStyle: 'none',
+});
+
+const titleStyle = css({
+  fontWeight: 'bold',
+});
+
+const detailsContentStyle = css({
+  marginBottom: `${24 / perRem}em`,
+});
+
+const priorities: Record<TeamRole, number> = {
+  'Lead PI (Core Leadership)': 1,
+  'Co-PI (Core Leadership)': 2,
+  'Collaborating PI': 3,
+  'Project Manager': 4,
+  'Key Personnel': 5,
+  'ASAP Staff': 6,
+  'Scientific Advisory Board': 7,
+};
+
+type UserTeamsTabbedCardProps = Pick<
+  ComponentProps<typeof TabbedCard>,
+  'description'
+> & {
+  userName: string;
+  isUserAlumni: boolean;
```

Comment:
Tom suggested we drop "is", "has" etc from variable naming due to the fact that we use TypeScript and thus it's easy to see what type that specific property has. So userAlumni would suffice.